cogs/StaffRequirements.py

```diff
@@ -55,10 +55,10 @@ async def checkreqs(self, ctx: commands.Context) -> None:
         ).build()
         await ctx.respond(embed=embed, ephemeral=True)
 
-        if (created_years_ago >= 1) and (joined_days_ago >= 30):
+        if (created_years_ago >= 1) and (joined_days_ago >= 30 or joined_years_ago >= 1):
             embed = EmbedBuilder(
                 title="Application Link",
-                description=f"Congratulations, {account.mention}! You meet the basic requirements for staff. Please fill out the application form [here](https://goo.gl/forms/Z3mVQwLdiNZcKHx52).",
+                description=f"Congratulations, {account.mention}! You meet the basic requirements for staff. Please fill out the application form [here]({getenv('staff_google_form')}).",
```

Comment:
It's not handy to hard code something that isn't supposed to be seen without the correct permissions. So I moved it to the env.
(It'd be handy to change the link as well because anyone will be able to see the old link)